syncd/syncd.cpp

```diff
@@ -1927,6 +1946,64 @@ sai_status_t processGetStatsEvent(
     return status;
 }
 
+sai_status_t processClearStatsEvent(
+        _In_ const swss::KeyOpFieldsValuesTuple &kco)
+{
+    SWSS_LOG_ENTER();
+
+    const std::string &key = kfvKey(kco);
+    const std::string &str_object_type = key.substr(0, key.find(":"));
+    const std::string &str_object_id = key.substr(key.find(":") + 1);
+
+    sai_object_id_t object_id;
+    sai_deserialize_object_id(str_object_id, object_id);
+    sai_object_id_t rid;
+    sai_status_t status = SAI_STATUS_FAILURE;
+    std::vector<swss::FieldValueTuple> fvTuples;
+    try
```

Comment:
i already added function for that:
https://github.com/Azure/sonic-sairedis/pull/464/files#diff-91b3e380b7f6f4a0a048ca8296a12e0dR631
which can be reused here

Reply:
Done